### PR TITLE
Bbtag update phase1 patch

### PR DESCRIPTION
**Pull request by Titansmasher**
Fixes for negative number comparison. I know its not ideal but its the best I could think of ;-;
Fix some other small issues.
Enable embeds in DM's

### CODE REVIEW

File: src/tags/dm.js [hunk 1]
```diff
@@ -13,8 +13,9 @@ const Builder = require('../structures/TagBuilder'),
 module.exports =
     Builder.CCommandTag('dm')
         .requireStaff()
-        .withArgs(a => [a.require('user'), a.require('message')])
-        .withDesc('DMs `user` the given `message`. You may only send one DM per execution. Requires author to be staff, and the user to be on the current guild.'
+        .withArgs(a => [a.require('user'), a.require([a.optional('message'), a.optional('embed')])])
+        .withDesc('DMs `user` the given `message` and `embed`. Atleast one of `message` and `embed` must be provided. ' +
```
Comment (Ratismal) [inline src/tags/dm.js]: Should say "At least"

File: src/tags/edit.js [hunk 2]
```diff
@@ -15,10 +15,11 @@ module.exports =
         .withArgs(a => [a.optional('channelId'), a.require('messageId'), a.require([a.optional('text'), a.optional('embed')])])
         .withDesc('Edits `messageId` in `channelId` to say `text` or `embed`. ' +
             'Atleast one of `text` and `embed` is required. ' +
-            'If `channelId` is not supplied, it defaults to the current channel. ' +
+            'If `channelId` is not supplied, it defaults to the current channel.\n' +
+            'Please note that `embed` is the JSON for an embed object, dont put the `{embed}` subtag there, as nothing will show.\n'+
```
Comment (Ratismal) [inline src/tags/edit.js]: Should be "don't"

File: src/tags/edit.js [hunk 3]
```diff
@@ -15,10 +15,11 @@ module.exports =
         .withArgs(a => [a.optional('channelId'), a.require('messageId'), a.require([a.optional('text'), a.optional('embed')])])
         .withDesc('Edits `messageId` in `channelId` to say `text` or `embed`. ' +
             'Atleast one of `text` and `embed` is required. ' +
-            'If `channelId` is not supplied, it defaults to the current channel. ' +
+            'If `channelId` is not supplied, it defaults to the current channel.\n' +
+            'Please note that `embed` is the JSON for an embed object, dont put the `{embed}` subtag there, as nothing will show.\n'+ 
             'Only messages created by the bot may be edited')
         .withExample(
-            'A message got edited: {edit;111111111111111111;New content}',
+            'A message got edited: {edit;111111111111111111;New content;{buildembed;title:Youre cool}}',
```
Comment (Ratismal) [inline src/tags/edit.js]: Should be "you're"

File: src/tags/send.js [hunk 4]
```diff
@@ -14,33 +14,37 @@ module.exports =
...
         .withArgs(a => [a.require('channel'), a.require([a.optional('message'), a.optional('embed')])])
         .withDesc('Sends `message` and `embed` to `channel`, and returns the message ID. `channel` is either an ID or channel mention. ' +
-            'At least one out of `message` and `embed` must be supplied')
+            'At least one out of `message` and `embed` must be supplied.\n' +
+            'Please note that `embed` is the JSON for an embed object, dont put the `{embed}` subtag there, as nothing will show')
         .withExample(
-            '{send;#channel;Hello!}',
-            '1111111111111111111\nIn #channel: Hello!'
+            '{send;#channel;Hello!;{buildembed;title:Youre cool}}',
+            '1111111111111111111\nIn #channel: Hello!\nEmbed: Youre cool'
```
Comment (Ratismal) [inline src/tags/send.js]: Should be "You're"

File: src/tags/edit.js (hunk 3 above)
Comment (Titansmasher) [reply src/tags/edit.js]: `You're` or `you're`? consistency please cat ;)

File: src/tags/send.js (hunk 4 above)
Comment (Titansmasher) [reply src/tags/send.js]: `You're` or `you're`? consistency please cat ;)

**Pull request merged**